lambda_functions/source/register-organization/lambda.py

```diff
@@ -87,6 +87,38 @@ def get_active_regions(AWS_REGION):
         return my_regions
     except Exception as e:
         return e
+
+def get_ssm_regions():
+    try:
+        supported_regions = [
+            "af-south-1",
+            "ap-east-1",
+            "ap-northeast-1",
+            "ap-northeast-2",
+            "ap-south-1",
+            "ap-southeast-1",
+            "ap-southeast-2",
+            "ca-central-1",
+            "eu-central-1",
+            "eu-north-1",
+            "eu-south-1",
+            "eu-west-1",
+            "eu-west-2",
+            "eu-west-3",
+            "me-south-1",
+            "sa-east-1",
+            "us-east-1",
+            "us-east-2",
+            "us-west-1",
+            "us-west-2"
+        ]
+        ssm_regions = []
+        for supported_region in supported_regions:
+            if supported_region not in EXCLUDE_REGIONS:
```

Comment:
fixed the function to account for opted-in regions.

for supporting new regions as they are added I'm not sure how else we would provide a new 'release' without the customer updating the stack.  This will not be constant though, as the regions our SSM Distributor supports are not going to change often.

Reply:
That's correct. We can't avoid update stack. However, the failure due to non-operable region can be gated with this. Thanks